util/src/main/scala/scala/scalanative/package.scala

```diff
@@ -0,0 +1,12 @@
+package scala
+import scala.collection.mutable
+import scala.language.implicitConversions
+
+package object scalanative {
+  implicit def bufferToSeq[T](
+      buf: mutable.UnrolledBuffer[T]): collection.Seq[T] =
+    buf.toSeq
+
+  implicit def seqToImmutableSeq[T](
+      seq: collection.Seq[T]): collection.immutable.Seq[T] = seq.toIndexedSeq
```

Comment:
We really should not have package-level implicit conversions like that. We should just explicitly use `.toSeq` and `.toIndexedSeq` at the call sites where required.